### PR TITLE
refactor code

### PR DESCRIPTION
**Pull request by hpn33**
change code style for readability

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
-		client.commands.get('search').execute(message, args);
-	} else if (command == 'help') {
-		client.commands.get('help').execute(message, args);
-	} else if (command == 'read') {
-		client.commands.get('read').execute(message, args);
-	} else if (command == 'random') {
-		client.commands.get('random').execute(message, args);
-	} else if (command == 'tags') {
-		client.commands.get('tags').execute(message, args);
+	if (client.commands.has(command)) {
```
Comment (crackheadakira) [inline bot.js:37]: How would more than one command be added? ( I do not have any coding experience, I used a lot of Stackoverflow for this bot )

**Review by crackheadakira — CHANGES_REQUESTED** [review]
I really appreciate the effort you put into this as I wrote this bot without having any prior coding experience meaning the code would be hard to read.

Before I merge your fork I would like the wEmbed ( warning embed for Discord ToS ) to also be implemented, along with the commands in bot.js as I tried before myself but I couldn't figure out how. I could only get one command working

File: bot.js (hunk 1 above)
Comment (hpn33) [reply bot.js:37]: you first read the files and
collect on `client.commands`

at last just check the command exist on `client.commands`
if was execute
`if (client.commands.has(command)) {`

and for adding command need make a new commands file on commands folder

or you want do more than one command at time

Comment (hpn33) [reply bot.js:37]: and for warning I do not have complete experience with develop on "discord".

**Pull request merged**